prog-template.c

```diff
@@ -383,9 +383,9 @@ void UDPsendSensor(int UDP_sockfd, struct sockaddr_in servaddr, long double T, d
 
 	// Accelerometer
 	robosar_fms_Accelerometer proto_accel_data;
-	proto_accel_data.acc_x = velo_cmd.V; // send received lin vel cmd
-	proto_accel_data.acc_y = velo_cmd.W; // send received ang vel cmd
-	proto_accel_data.acc_z = override_flag;
+	proto_accel_data.acc_x = acc_X;
+	proto_accel_data.acc_y = acc_Y;
+	proto_accel_data.acc_z = acc_Z;
```

Comment:
Was this previously set to the velocity commands instead for debugging purposes?

Reply:
Yes, but the debugging has been resolved, so I wanted to set them back